bionic/persistence.py

```diff
@@ -501,6 +515,21 @@ def find_entry(self, query):
             value_hash=metadata_record.value_hash,
         )
 
+    def list_items(self):
+        metadata_urls = [
+            url for url in self._fs.search(self.root_url) if url.endswith(".yaml")
```

Comment:
Not in this PR but we should make filesystem search relative to the `root_url`. Then we can pass it `.` instead.

Reply:
Yep, that would work too. (I guess we'd just have to do the URL joining ourselves in the GCS filesystem.)